imsim/sky_model.py

```diff
@@ -108,6 +115,50 @@ def __call__(self, x, y):
         value at the CCD center.
         """
         return (self.a*x + self.b*y + self.c)/self.sky_level_center
+    
+class CCD_Fringe:
+    """
+    Class generates normalized fringing map. 
+    
+    The function call operator returns the normlized fringing level
+    as a function of pixel coordinates relative to the value at the 
+    CCD center.
+    """
+    def __init__(self):
+        
+        # Load the 4k x 4k normalized map
+        fringing_filename = os.path.join(data_dir, 'fringing_data',
+                                    'e2v-321-fringe-sim-norm-center.fits.gz')
+        
+        fringe_im = fits.open(fringing_filename)[0].data
+        
+        # Interpolate
+        x = np.linspace(0,fringe_im.shape[-1]-1,fringe_im.shape[-1],dtype= int)
+        y = np.linspace(0, fringe_im.shape[0]-1,fringe_im.shape[0],dtype = int )
```

Comment:
Probably worth adding spaces after the commas here.  I'm not a stickler for pep8 style, but these two lines are a bit hard to read without them.